### ISSUE TITLE
Documentation and a live example needed

### ISSUE THREAD

**Issue by Emasoft** [ericwilligers/svg-animation#34]
This is a very important library for SVG animators like me, thank you. But please add some documentation and a live example.

**Comment by joe312213** [comment]
yes please!

**Comment by jaminroe** [comment]
Yes #1 issue. I don't know JavaScript since I'm a designer. I hope this will let me still use SMIL the way it was. :( thank you!

**Comment by jpiv** [comment]
bump

**Comment by Emasoft** [comment]
The author of the Flash2Svg tool, that uses SMIL to export flash animations, published an usage example:

Without SMIL polyfill:
http://www.tbyrne.org/staging/smil-polyfill/tears.htm

With SMIL polyfill:
http://www.tbyrne.org/staging/smil-polyfill/tears_polyfill.htm
